central/enrichment/singleton.go

```diff
@@ -51,8 +55,20 @@ func initialize() {
 		signatureIntegrationDataStore.Singleton().GetAllSignatureIntegrations)
 	ne = nodeEnricher.New(nodeCVESuppressor, metrics.CentralSubsystem)
 	en = New(datastore.Singleton(), ie)
+	ad = imageIntegrationDS.Singleton()
```

Comment:
nit: initialize this closer to where it's used (ie below where the `newManager` is made)